.github/workflows/deploy.yml

```diff
@@ -21,4 +24,13 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install -g yarn
-    - run: ./deploy.sh
+    - run: yarn
+    - run: git config --global user.email "actions@codysherman.com"
+    - run: git config --global user.name "GitHub Actions"
+    - run: ./deploy.sh ${{ secrets.GITHUB_TOKEN }} ${{ github.repository }}
+    - run: >-
+           curl -L -X POST -w '%{http_code}\n'
+           -H "Content-Type: application/json"
+           -H "Authorization: token ${{ secrets.DEPLOY_KEY }}"
```

Comment:
I think whatever is stored in the DEPLOY_KEY field is still a PAT, right? I don't think you could make that API request with a deploy key since it's a ssh key, versus a PAT which is for HTTP requests.